src/advanced/node-management.md

```diff
@@ -81,6 +81,8 @@ address formatted as 4 octets separated by dots (for example, `10.10.0.1`).
   ID of the validator that created an approved block proposal
 - **schema_version**: integer  
   Information schema version. Currently, `0`
+- **state_hash**: Hash  
+  Hash of the current Exonum state after applying transactions in the block  
```

Comment:
Nitpick: Would you mind linking to [the definition of Exonum state from the glossary](https://exonum.com/doc/glossary/#blockchain-state)?